### PR TITLE
Install on Azure AKS

### PR DESCRIPTION
**Pull request by simonferquel**
Based on #18 (AKS for now requires to skip liveness probes)

This introduces instruction to install on Azure AKS

### CODE REVIEW

File: docs/install-on-aks.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Install on Azure AKS
+
+## Pre-requisites
+- To install Compose on Kubernetes on Azure AKS, you must create an AKS cluster with RBAC enabled.
+- To install etcd using those instructions, you must have Helm on your client environment
+- Download the Compose on Kubernetes installer
```
Comment (silvin-lubecki) [inline docs/install-on-aks.md]: Can you add a link for this? Or maybe just a `go get` is enough?

File: docs/install-on-aks.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- Run `kubectl apply -f compose-etcd.yaml`
+- This should bring an etcd cluster in the compose namespace
+
+**Note: this cluster configuration is really naive and does does not use mutual TLS to authenticate application accessing the data. For enabling mutual TLS, please refer to https://github.com/coreos/etcd-operator** 
+
+## Deploy Compose on Kubernetes
+
+run `installer-[darwin|linux|windows.exe] -namespace=compose -etcd-servers=http://compose-etcd-client:2379 -tag=v0.4.16 -skip-liveness-probes=true`
+
+**Note: There is currently an issue with AKS where the API Server Liveness Probe always failes due to a TLS issue. We'll get in touch with Microsoft to try to solve the issue in the future.**
```
Comment (silvin-lubecki) [inline docs/install-on-aks.md]: typo: failes -> fails

File: docs/install-on-aks.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  version: "3.2.13"
+```
+- Run `kubectl apply -f compose-etcd.yaml`
+- This should bring an etcd cluster in the compose namespace
+
+**Note: this cluster configuration is really naive and does does not use mutual TLS to authenticate application accessing the data. For enabling mutual TLS, please refer to https://github.com/coreos/etcd-operator** 
+
+## Deploy Compose on Kubernetes
+
+run `installer-[darwin|linux|windows.exe] -namespace=compose -etcd-servers=http://compose-etcd-client:2379 -tag=v0.4.16 -skip-liveness-probes=true`
```
Comment (silvin-lubecki) [inline docs/install-on-aks.md]: run -> Run

**Comment by simonferquel** [comment]
Seems that we don't vendor gcp client code in docker cli. @silvin-lubecki can you confirm?

If it is indeed the case, we'll make a pr on docker cli and provide instructions to build a gcp-compatible docker cli until it makes it in next release.

**Comment by silvin-lubecki** [comment]
@simonferquel : @jdrouet handled that in #24 .

**Pull request merged**